tests/test_app_percy.py

```diff
@@ -15,7 +16,7 @@
 from percy.metadata import AndroidMetadata
 from tests.mocks.mock_methods import android_capabilities, ios_capabilities
 
-
+@patch.dict(os.environ, {"PERCY_DISABLE_REMOTE_UPLOADS": "true"})
```

Comment:
What tests cover the cases when this is not set or set to false ? shouldnt we have an e2e test for that as well ?

Reply:
added e2e tests for when disable upload is not given.